
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="A live review and exploration of Spatial Omics literature and data.">
      
      
      
        <link rel="canonical" href="https://marta-seq.github.io/solr/book/references/">
      
      
        <link rel="prev" href="../chapter3/">
      
      
        <link rel="next" href="../../datasets/">
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.14">
    
    
      
        <title>References - Spatial Omics Live Review</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="light-blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Spatial Omics Live Review" class="md-header__button md-logo" aria-label="Spatial Omics Live Review" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Spatial Omics Live Review
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              References
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="light-blue"  aria-hidden="true"  type="radio" name="__palette" id="__palette_0">
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../" class="md-tabs__link">
          
  
  
  Book

        </a>
      </li>
    
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../../datasets/" class="md-tabs__link">
        
  
  
    
  
  Spatial Omics Datasets

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../../methods/" class="md-tabs__link">
        
  
  
    
  
  Spatial Omics Methods

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../../applications/" class="md-tabs__link">
        
  
  
    
  
  Spatial Omics Applications

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Spatial Omics Live Review" class="md-nav__button md-logo" aria-label="Spatial Omics Live Review" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    Spatial Omics Live Review
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
      
        
        
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Book
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Book
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter1/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chapter 1
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chapter 2
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter3/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chapter 3
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    References
    
  </span>
  

      </a>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../datasets/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Spatial Omics Datasets
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../methods/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Spatial Omics Methods
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../applications/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Spatial Omics Applications
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>References</h1>

<p><strong>References</strong></p>
<p>1. Hanahan D. Hallmarks of Cancer: New Dimensions. Cancer Discov. 2022
Jan 1;12(1):31--46.</p>
<p>2. Hanahan D, Weinberg RA. Hallmarks of Cancer: The Next Generation.
Cell. 2011 Mar;144(5):646--74.</p>
<p>3. Sung H, Ferlay J, Siegel RL, Laversanne M, Soerjomataram I, Jemal A,
et al. Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence
and Mortality Worldwide for 36 Cancers in 185 Countries. CA Cancer J
Clin. 2021 May;71(3):209--49.</p>
<p>4. World Health Organization. Cancer Fact Sheet. [Internet]. 2021.
Available from: https://gco.iarc.fr/today/en</p>
<p>5. Ferlay J, Colombet M, Soerjomataram I, Parkin DM, Piñeros M, Znaor
A, et al. Cancer statistics for the year 2020: An overview. Int J
Cancer. 2021 Aug 15;149(4):778--89.</p>
<p>6. Dagogo-Jack I, Shaw AT. Tumour heterogeneity and resistance to
cancer therapies. Nat Rev Clin Oncol. 2018 Feb;15(2):81--94.</p>
<p>7. Bedard PL, Hansen AR, Ratain MJ, Siu LL. Tumour heterogeneity in the
clinic. Nature. 2013 Sep 19;501(7467):355--64.</p>
<p>8. Anderson NM, Simon MC. The tumor microenvironment. Curr Biol. 2020
Aug;30(16):R921--5.</p>
<p>9. Farc O, Cristea V. An overview of the tumor microenvironment, from
cells to complex networks (Review). Exp Ther Med. 2020 Nov 26;21(1):96.</p>
<p>10. Fridman WH, Zitvogel L, Sautès--Fridman C, Kroemer G. The immune
contexture in cancer prognosis and treatment. Nat Rev Clin Oncol. 2017
Dec;14(12):717--34.</p>
<p>11. Roelands J, Kuppen P, Vermeulen L, Maccalli C, Decock J, Wang E, et
al. Immunogenomic Classification of Colorectal Cancer and Therapeutic
Implications. Int J Mol Sci. 2017 Oct 24;18(10):2229.</p>
<p>12. Pagès F, Mlecnik B, Marliot F, Bindea G, Ou FS, Bifulco C, et al.
International validation of the consensus Immunoscore for the
classification of colon cancer: a prognostic and accuracy study. The
Lancet. 2018 May;391(10135):2128--39.</p>
<p>13. Galon J, Costes A, Sanchez-Cabo F, Kirilovsky A, Mlecnik B,
Lagorce-Pagès C, et al. Type, Density, and Location of Immune Cells
Within Human Colorectal Tumors Predict Clinical Outcome. Science. 2006
Sep 29;313(5795):1960--4.</p>
<p>14. Yang D, Liu J, Qian H, Zhuang Q. Cancer-associated fibroblasts:
from basic science to anticancer therapy. Exp Mol Med. 2023 Jul
3;55(7):1322--32.</p>
<p>15. Sahai E, Astsaturov I, Cukierman E, DeNardo DG, Egeblad M, Evans
RM, et al. A framework for advancing our understanding of
cancer-associated fibroblasts. Nat Rev Cancer. 2020 Mar;20(3):174--86.</p>
<p>16. The Cancer Genome Atlas Research Network, Weinstein JN, Collisson
EA, Mills GB, Shaw KRM, Ozenberger BA, et al. The Cancer Genome Atlas
Pan-Cancer analysis project. Nat Genet. 2013 Oct;45(10):1113--20.</p>
<p>17. Hoadley KA, Yau C, Hinoue T, Wolf DM, Lazar AJ, Drill E, et al.
Cell-of-Origin Patterns Dominate the Molecular Classification of 10,000
Tumors from 33 Types of Cancer. Cell. 2018 Apr;173(2):291-304.e6.</p>
<p>18. The Cancer Genome Atlas Network. Comprehensive molecular
characterization of human colon and rectal cancer. Nature. 2012 Jul
19;487(7407):330--7.</p>
<p>19. Nunes L, Li F, Wu M, Luo T, Hammarström K, Torell E, et al.
Prognostic genome and transcriptome signatures in colorectal cancers.
Nature. 2024 Sep 5;633(8028):137--46.</p>
<p>20. Li Y. Modern epigenetics methods in biological research. Methods
San Diego Calif. 2021 Mar;187:104--13.</p>
<p>21. Chen X, Xu H, Shu X, Song CX. Mapping epigenetic modifications by
sequencing technologies. Cell Death Differ. 2025 Jan;32(1):56--65.</p>
<p>22. Baylin SB, Jones PA. A decade of exploring the cancer epigenome ---
biological and translational implications. Nat Rev Cancer. 2011
Oct;11(10):726--34.</p>
<p>23. Uhlén M, Fagerberg L, Hallström BM, Lindskog C, Oksvold P,
Mardinoglu A, et al. Tissue-based map of the human proteome. Science.
2015 Jan 23;347(6220):1260419.</p>
<p>24. Suhre K, McCarthy MI, Schwenk JM. Genetics meets proteomics:
perspectives for large population-based studies. Nat Rev Genet. 2021
Jan;22(1):19--37.</p>
<p>25. Aebersold R, Mann M. Mass-spectrometric exploration of proteome
structure and function. Nature. 2016 Sep;537(7620):347--55.</p>
<p>26. Schmidt DR, Patel R, Kirsch DG, Lewis CA, Vander Heiden MG,
Locasale JW. Metabolomics in cancer research and emerging applications
in clinical oncology. CA Cancer J Clin. 2021 Jul;71(4):333--58.</p>
<p>27. Aldridge S, Teichmann SA. Single cell transcriptomics comes of age.
Nat Commun. 2020 Aug 27;11(1):4307.</p>
<p>28. Kinker GS, Greenwald AC, Tal R, Orlova Z, Cuoco MS, McFarland JM,
et al. Pan-cancer single-cell RNA-seq identifies recurring programs of
cellular heterogeneity. Nat Genet. 2020 Nov;52(11):1208--18.</p>
<p>29. Mansuri MS, Williams K, Nairn AC. Uncovering biology by single-cell
proteomics. Commun Biol. 2023 Apr 8;6(1):381.</p>
<p>30. Single-cell proteomics: challenges and prospects. Nat Methods. 2023
Mar;20(3):317--8.</p>
<p>31. Setayesh SM, Ndacayisaba LJ, Rappard KE, Hennes V, Rueda LYM, Tang
G, et al. Targeted single-cell proteomic analysis identifies new liquid
biopsy biomarkers associated with multiple myeloma. Npj Precis Oncol.
2023 Sep 18;7(1):95.</p>
<p>32. Seydel C. Single-cell metabolomics hits its stride. Nat Methods.
2021 Dec;18(12):1452--6.</p>
<p>33. Hu T, Allam M, Cai S, Henderson W, Yueh B, Garipcan A, et al.
Single-cell spatial metabolomics with cell-type specific protein
profiling for tissue systems biology. Nat Commun. 2023 Dec
13;14(1):8260.</p>
<p>34. Clark K, Vendt B, Smith K, Freymann J, Kirby J, Koppel P, et al.
The Cancer Imaging Archive (TCIA): Maintaining and Operating a Public
Information Repository. J Digit Imaging. 2013 Dec;26(6):1045--57.</p>
<p>35. Liu L, Chen A, Li Y, Mulder J, Heyn H, Xu X. Spatiotemporal omics
for biology and medicine. Cell. 2024 Aug;187(17):4488--519.</p>
<p>36. Palla G, Fischer DS, Regev A, Theis FJ. Spatial components of
molecular tissue biology. Nat Biotechnol. 2022 Mar;40(3):308--18.</p>
<p>37. Lundberg E, Borner GHH. Spatial proteomics: a powerful discovery
tool for cell biology. Nat Rev Mol Cell Biol. 2019 May;20(5):285--302.</p>
<p>38. Jing S yu, Wang H qi, Lin P, Yuan J, Tang Z xuan, Li H. Quantifying
and interpreting biologically meaningful spatial signatures within tumor
microenvironments. Npj Precis Oncol. 2025 Mar 11;9(1):68.</p>
<p>39. Hickey JW, Neumann EK, Radtke AJ, Camarillo JM, Beuschel RT,
Albanese A, et al. Spatial mapping of protein composition and tissue
organization: a primer for multiplexed antibody-based imaging. Nat
Methods. 2022 Mar;19(3):284--95.</p>
<p>40. Lin JR, Izar B, Wang S, Yapp C, Mei S, Shah PM, et al. Highly
multiplexed immunofluorescence imaging of human tissues and tumors using
t-CyCIF and conventional optical microscopes. eLife. 2018 Jul
11;7:e31657.</p>
<p>41. Radtke AJ, Chu CJ, Yaniv Z, Yao L, Marr J, Beuschel RT, et al.
IBEX: an iterative immunolabeling and chemical bleaching method for
high-content imaging of diverse tissues. Nat Protoc. 2022
Feb;17(2):378--401.</p>
<p>42. Goltsev Y, Samusik N, Kennedy-Darling J, Bhate S, Hale M, Vazquez
G, et al. Deep Profiling of Mouse Splenic Architecture with CODEX
Multiplexed Imaging. Cell. 2018 Aug;174(4):968-981.e15.</p>
<p>43. Black S, Phillips D, Hickey JW, Kennedy-Darling J, Venkataraaman
VG, Samusik N, et al. CODEX multiplexed tissue imaging with
DNA-conjugated antibodies. Nat Protoc. 2021 Aug;16(8):3802--35.</p>
<p>44. Saka SK, Wang Y, Kishi JY, Zhu A, Zeng Y, Xie W, et al.
Immuno-SABER enables highly multiplexed and amplified protein imaging in
tissues. Nat Biotechnol. 2019 Sep;37(9):1080--90.</p>
<p>45. Giesen C, Wang HAO, Schapiro D, Zivanovic N, Jacobs A, Hattendorf
B, et al. Highly multiplexed imaging of tumor tissues with subcellular
resolution by mass cytometry. Nat Methods. 2014 Apr;11(4):417--22.</p>
<p>46. Keren L, Bosse M, Thompson S, Risom T, Vijayaragavan K, McCaffrey
E, et al. MIBI-TOF: A multiplexed imaging platform relates cellular
phenotypes and tissue structure. Sci Adv. 2019 Oct 11;5(10):eaax5851.</p>
<p>47. Asp M, Bergenstråhle J, Lundeberg J. Spatially Resolved
Transcriptomes---Next Generation Tools for Tissue Exploration.
BioEssays. 2020 Oct;42(10):1900221.</p>
<p>48. Du J, Yang YC, An ZJ, Zhang MH, Fu XH, Huang ZF, et al. Advances in
spatial transcriptomics and related data analysis strategies. J Transl
Med. 2023 May 18;21(1):330.</p>
<p>49. Moses L, Pachter L. Museum of spatial transcriptomics. Nat Methods.
2022 May;19(5):534--46.</p>
<p>50. Chen KH, Boettiger AN, Moffitt JR, Wang S, Zhuang X. Spatially
resolved, highly multiplexed RNA profiling in single cells. Science.
2015 Apr 24;348(6233):aaa6090.</p>
<p>51. Eng CHL, Lawson M, Zhu Q, Dries R, Koulena N, Takei Y, et al.
Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+.
Nature. 2019 Apr;568(7751):235--9.</p>
<p>52. He S, Bhatt R, Brown C, Brown EA, Buhr DL, Chantranuvatana K, et
al. High-plex Multiomic Analysis in FFPE at Subcellular Level by Spatial
Molecular Imaging [Internet]. Genomics; 2021 [cited 2024 Nov 12].
Available from: http://biorxiv.org/lookup/doi/10.1101/2021.11.03.467020</p>
<p>53. He S, Bhatt R, Brown C, Brown EA, Buhr DL, Chantranuvatana K, et
al. High-plex imaging of RNA and proteins at subcellular resolution in
fixed tissue by spatial molecular imaging. Nat Biotechnol. 2022
Dec;40(12):1794--806.</p>
<p>54. Janesick A, Shelansky R, Gottscho AD, Wagner F, Williams SR,
Rouault M, et al. High resolution mapping of the tumor microenvironment
using integrated single-cell, spatial and in situ analysis. Nat Commun.
2023 Dec 19;14(1):8353.</p>
<p>55. Wang X, Allen WE, Wright MA, Sylwestrak EL, Samusik N, Vesuna S, et
al. Three-dimensional intact-tissue sequencing of single-cell
transcriptional states. Science. 2018 Jul 27;361(6400):eaat5691.</p>
<p>56. Lee JH, Daugharthy ER, Scheiman J, Kalhor R, Ferrante TC, Terry R,
et al. Fluorescent in situ sequencing (FISSEQ) of RNA for gene
expression profiling in intact cells and tissues. Nat Protoc. 2015
Mar;10(3):442--58.</p>
<p>57. Ståhl PL, Salmén F, Vickovic S, Lundmark A, Navarro JF, Magnusson
J, et al. Visualization and analysis of gene expression in tissue
sections by spatial transcriptomics. Science. 2016 Jul;353(6294):78--82.</p>
<p>58. Rodriques SG, Stickels RR, Goeva A, Martin CA, Murray E, Vanderburg
CR, et al. Slide-seq: A scalable technology for measuring genome-wide
expression at high spatial resolution. Science. 2019 Mar
29;363(6434):1463--7.</p>
<p>59. Stickels RR, Murray E, Kumar P, Li J, Marshall JL, Di Bella DJ, et
al. Highly sensitive spatial transcriptomics at near-cellular resolution
with Slide-seqV2. Nat Biotechnol. 2021 Mar;39(3):313--9.</p>
<p>60. Aichler M, Walch A. MALDI Imaging mass spectrometry: current
frontiers and perspectives in pathology research and practice. Lab
Invest. 2015 Apr;95(4):422--31.</p>
<p>61. Stoeckli M, Chaurand P, Hallahan DE, Caprioli RM. Imaging mass
spectrometry: A new technology for the analysis of protein expression in
mammalian tissues. Nat Med. 2001 Apr;7(4):493--6.</p>
<p>62. Merritt CR, Ong GT, Church SE, Barker K, Danaher P, Geiss G, et al.
Multiplex digital spatial profiling of proteins and RNA in fixed tissue.
Nat Biotechnol. 2020 May;38(5):586--99.</p>
<p>63. Liu Y, Yang M, Deng Y, Su G, Enninful A, Guo CC, et al.
High-Spatial-Resolution Multi-Omics Sequencing via Deterministic
Barcoding in Tissue. Cell. 2020 Dec;183(6):1665-1681.e18.</p>
<p>64. Liu Y, DiStasio M, Su G, Asashima H, Enninful A, Qin X, et al.
High-plex protein and whole transcriptome co-mapping at cellular
resolution with spatial CITE-seq. Nat Biotechnol. 2023
Oct;41(10):1405--9.</p>
<p>65. Nunes JB, Ijsselsteijn ME, Abdelaal T, Ursem R, Van Der Ploeg M,
Giera M, et al. Integration of mass cytometry and mass spectrometry
imaging for spatially resolved single-cell metabolic profiling. Nat
Methods. 2024 Oct;21(10):1796--800.</p>
<p>66. Vicari M, Mirzazadeh R, Nilsson A, Shariatgorji R, Bjärterot P,
Larsson L, et al. Spatial multimodal analysis of transcriptomes and
metabolomes in tissues. Nat Biotechnol. 2024 Jul;42(7):1046--50.</p>
<p>67. Engblom C, Lundeberg J. Putting cancer immunotherapy into spatial
context in the clinic. Nat Biotechnol. 2025 Apr;43(4):471--6.</p>
<p>68. Allam M, Cai S, Coskun AF. Multiplex bioimaging of single-cell
spatial profiles for precision cancer diagnostics and therapeutics. Npj
Precis Oncol. 2020 May 1;4(1):11.</p>
<p>69. Hsieh WC, Budiarto BR, Wang YF, Lin CY, Gwo MC, So DK, et al.
Spatial multi-omics analyses of the tumor immune microenvironment. J
Biomed Sci. 2022 Nov 14;29(1):96.</p>
<p>70. Giesen C, Wang HAO, Schapiro D, Zivanovic N, Jacobs A, Hattendorf
B, et al. Highly multiplexed imaging of tumor tissues with subcellular
resolution by mass cytometry. Nat Methods. 2014 Apr;11(4):417--22.</p>
<p>71. Engblom C, Thrane K, Lin Q, Andersson A, Toosi H, Chen X, et al.
Spatial transcriptomics of B cell and T cell receptors reveals
lymphocyte clonal dynamics. Science. 2023 Dec 8;382(6675):eadf8486.</p>
<p>72. Baharlou H, Canete NP, Cunningham AL, Harman AN, Patrick E. Mass
Cytometry Imaging for the Study of Human Diseases---Applications and
Data Analysis Strategies. Front Immunol. 2019 Nov 14;10:2657.</p>
<p>73. Baranski A, Milo I, Greenbaum S, Oliveria JP, Mrdjen D, Angelo M,
et al. MAUI (MBI Analysis User Interface)---An image processing pipeline
for Multiplexed Mass Based Imaging. Schneidman-Duhovny D, editor. PLOS
Comput Biol. 2021 Apr 19;17(4):e1008887.</p>
<p>74. Chevrier S, Crowell HL, Zanotelli VRT, Engler S, Robinson MD,
Bodenmiller B. Compensation of Signal Spillover in Suspension and
Imaging Mass Cytometry. Cell Syst. 2018 May;6(5):612-620.e5.</p>
<p>75. Milosevic V. Different approaches to Imaging Mass Cytometry data
analysis. Yu G, editor. Bioinforma Adv. 2023 Jan 5;3(1):vbad046.</p>
<p>76. Wang YJ, Traum D, Schug J, Gao L, Liu C, Atkinson MA, et al.
Multiplexed In Situ Imaging Mass Cytometry Analysis of the Human
Endocrine Pancreas and Immune System in Type 1 Diabetes. Cell Metab.
2019 Mar;29(3):769-783.e4.</p>
<p>77. Windhager J, Zanotelli VRT, Schulz D, Meyer L, Daniel M,
Bodenmiller B, et al. An end-to-end workflow for multiplexed image
processing and analysis. Nat Protoc. 2023 Nov;18(11):3565--613.</p>
<p>78. Keren L, Bosse M, Marquez D, Angoshtari R, Jain S, Varma S, et al.
A Structured Tumor-Immune Microenvironment in Triple Negative Breast
Cancer Revealed by Multiplexed Ion Beam Imaging. Cell. 2018
Sep;174(6):1373-1387.e19.</p>
<p>79. Rendeiro AF, Ravichandran H, Bram Y, Chandar V, Kim J, Meydan C, et
al. The spatial landscape of lung pathology during COVID-19 progression.
Nature. 2021 May 27;593(7860):564--9.</p>
<p>80. Berg S, Kutra D, Kroeger T, Straehle CN, Kausler BX, Haubold C, et
al. ilastik: interactive machine learning for (bio)image analysis. Nat
Methods. 2019 Dec;16(12):1226--32.</p>
<p>81. Ijsselsteijn ME, Somarakis A, Lelieveldt BPF, Höllt T, De Miranda
NFCC. Semi‐automated background removal limits data loss and normalizes
imaging mass cytometry data. Cytometry A. 2021 Dec;99(12):1187--97.</p>
<p>82. Lu P, Oetjen KA, Bender DE, Ruzinova MB, Fisher DAC, Shim KG, et
al. IMC-Denoise: a content aware denoising pipeline to enhance Imaging
Mass Cytometry. Nat Commun. 2023 Mar 23;14(1):1601.</p>
<p>83. Krull A, Buchholz TO, Jug F. Noise2Void - Learning Denoising From
Single Noisy Images. In: 2019 IEEE/CVF Conference on Computer Vision and
Pattern Recognition (CVPR) [Internet]. Long Beach, CA, USA: IEEE; 2019
[cited 2025 Apr 9]. p. 2124--32. Available from:
https://ieeexplore.ieee.org/document/8954066/</p>
<p>84. Carpenter AE, Jones TR, Lamprecht MR, Clarke C, Kang IH, Friman O,
et al. CellProfiler: image analysis software for identifying and
quantifying cell phenotypes. Genome Biol. 2006 Oct 31;7(10):R100.</p>
<p>85. McQuin C, Goodman A, Chernyshev V, Kamentsky L, Cimini BA, Karhohs
KW, et al. CellProfiler 3.0: Next-generation image processing for
biology. Misteli T, editor. PLOS Biol. 2018 Jul 3;16(7):e2005970.</p>
<p>86. Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M,
Pietzsch T, et al. Fiji: an open-source platform for biological-image
analysis. Nat Methods. 2012 Jul;9(7):676--82.</p>
<p>87. Von Chamier L, Laine RF, Jukkala J, Spahn C, Krentzel D, Nehme E,
et al. Democratising deep learning for microscopy with ZeroCostDL4Mic.
Nat Commun. 2021 Apr 15;12(1):2276.</p>
<p>88. Greenwald NF, Miller G, Moen E, Kong A, Kagel A, Dougherty T, et
al. Whole-cell segmentation of tissue images with human-level
performance using large-scale data annotation and deep learning. Nat
Biotechnol. 2022 Apr;40(4):555--65.</p>
<p>89. Stringer C, Wang T, Michaelos M, Pachitariu M. Cellpose: a
generalist algorithm for cellular segmentation. Nat Methods. 2021
Jan;18(1):100--6.</p>
<p>90. Pachitariu M, Stringer C. Cellpose 2.0: how to train your own
model. Nat Methods. 2022 Dec;19(12):1634--41.</p>
<p>91. Lee MY, Bedia JS, Bhate SS, Barlow GL, Phillips D, Fantl WJ, et al.
CellSeg: a robust, pre-trained nucleus segmentation and pixel
quantification software for highly multiplexed fluorescence images. BMC
Bioinformatics. 2022 Dec;23(1):46.</p>
<p>92. Schapiro D, Sokolov A, Yapp C, Chen YA, Muhlich JL, Hess J, et al.
MCMICRO: a scalable, modular image-processing pipeline for multiplexed
tissue imaging. Nat Methods. 2022 Mar;19(3):311--5.</p>
<p>93. Baars MJD, Sinha N, Amini M, Pieterman-Bos A, Van Dam S, Ganpat
MMP, et al. MATISSE: a method for improved single cell segmentation in
imaging mass cytometry. BMC Biol. 2021 Dec;19(1):99.</p>
<p>94. Mandal S, Uhlmann V. SplineDist: Automated Cell Segmentation With
Spline Curves [Internet]. Bioinformatics; 2020 [cited 2025 Apr 19].
Available from: http://biorxiv.org/lookup/doi/10.1101/2020.10.27.357640</p>
<p>95. Schmidt U, Weigert M, Broaddus C, Myers G. Cell Detection with
Star-convex Polygons. 2018 [cited 2025 Apr 19]; Available from:
https://arxiv.org/abs/1806.03535</p>
<p>96. Petukhov V, Xu RJ, Soldatov RA, Cadinu P, Khodosevich K, Moffitt
JR, et al. Cell segmentation in imaging-based spatial transcriptomics.
Nat Biotechnol. 2022 Mar;40(3):345--54.</p>
<p>97. Park J, Choi W, Tiesmeyer S, Long B, Borm LE, Garren E, et al. Cell
segmentation-free inference of cell types from in situ transcriptomics
data. Nat Commun. 2021 Jun 10;12(1):3545.</p>
<p>98. Prabhakaran S. Sparcle: assigning transcripts to cells in
multiplexed images. Bateman A, editor. Bioinforma Adv. 2022 Jan
10;2(1):vbac048.</p>
<p>99. He Y, Tang X, Huang J, Ren J, Zhou H, Chen K, et al. ClusterMap for
multi-scale clustering analysis of spatial gene expression. Nat Commun.
2021 Oct 8;12(1):5909.</p>
<p>100. Si Y, Lee C, Hwang Y, Yun JH, Cheng W, Cho CS, et al. FICTURE:
scalable segmentation-free analysis of submicron-resolution spatial
transcriptomics. Nat Methods. 2024 Oct;21(10):1843--54.</p>
<p>101. Fu X, Lin Y, Lin DM, Mechtersheimer D, Wang C, Ameen F, et al.
BIDCell: Biologically-informed self-supervised learning for segmentation
of subcellular spatial transcriptomics data. Nat Commun. 2024 Jan
13;15(1):509.</p>
<p>102. Defard T, Laporte H, Ayan M, Soulier J, Curras-Alonso S, Weber C,
et al. A point cloud segmentation framework for image-based spatial
transcriptomics. Commun Biol. 2024 Jul 6;7(1):823.</p>
<p>103. Partel G, Wählby C. Spage2vec: Unsupervised representation of
localized spatial gene expression signatures. FEBS J. 2021
Mar;288(6):1859--70.</p>
<p>104. Qian X, Harris KD, Hauling T, Nicoloutsopoulos D, Muñoz-Manchado
AB, Skene N, et al. Probabilistic cell typing enables fine mapping of
closely related cell types in situ. Nat Methods. 2020 Jan;17(1):101--6.</p>
<p>105. Marco Salas S, Kuemmerle LB, Mattsson-Langseth C, Tismeyer S,
Avenel C, Hu T, et al. Optimizing Xenium In Situ data utility by quality
assessment and best-practice analysis workflows. Nat Methods. 2025
Apr;22(4):813--23.</p>
<p>106. Van Unen V, Höllt T, Pezzotti N, Li N, Reinders MJT, Eisemann E,
et al. Visual analysis of mass cytometry data by hierarchical stochastic
neighbour embedding reveals rare cell types. Nat Commun. 2017 Nov
23;8(1):1740.</p>
<p>107. Levine JH, Simonds EF, Bendall SC, Davis KL, Amir E ad D, Tadmor
MD, et al. Data-Driven Phenotypic Dissection of AML Reveals
Progenitor-like Cells that Correlate with Prognosis. Cell. 2015
Jul;162(1):184--97.</p>
<p>108. Van Gassen S, Callebaut B, Van Helden MJ, Lambrecht BN, Demeester
P, Dhaene T, et al. FlowSOM: Using self‐organizing maps for
visualization and interpretation of cytometry data. Cytometry A. 2015
Jul;87(7):636--45.</p>
<p>109. Quintelier K, Couckuyt A, Emmaneel A, Aerts J, Saeys Y, Van Gassen
S. Analyzing high-dimensional cytometry data using FlowSOM. Nat Protoc.
2021 Aug;16(8):3775--801.</p>
<p>110. Heumos L, Schaar AC, Lance C, Litinetskaya A, Drost F, Zappia L,
et al. Best practices for single-cell analysis across modalities. Nat
Rev Genet. 2023 Aug;24(8):550--72.</p>
<p>111. Blondel VD, Guillaume JL, Lambiotte R, Lefebvre E. Fast unfolding
of communities in large networks. J Stat Mech Theory Exp. 2008 Oct
1;2008(10):P10008.</p>
<p>112. Waltman L, Van Eck NJ. A smart local moving algorithm for
large-scale modularity-based community detection. Eur Phys J B. 2013
Nov;86(11):471.</p>
<p>113. Traag VA, Waltman L, Van Eck NJ. From Louvain to Leiden:
guaranteeing well-connected communities. Sci Rep. 2019 Mar 26;9(1):5233.</p>
<p>114. Domínguez Conde C, Xu C, Jarvis LB, Rainbow DB, Wells SB, Gomes T,
et al. Cross-tissue immune cell analysis reveals tissue-specific
features in humans. Science. 2022 May 13;376(6594):eabl5197.</p>
<p>115. Kang JB, Nathan A, Weinand K, Zhang F, Millard N, Rumker L, et al.
Efficient and precise single-cell reference atlas mapping with Symphony.
Nat Commun. 2021 Oct 7;12(1):5890.</p>
<p>116. Schapiro D, Jackson HW, Raghuraman S, Fischer JR, Zanotelli VRT,
Schulz D, et al. histoCAT: analysis of cell phenotypes and interactions
in multiplex image cytometry data. Nat Methods. 2017 Sep;14(9):873--6.</p>
<p>117. Schürch CM, Bhate SS, Barlow GL, Phillips DJ, Noti L, Zlobec I, et
al. Coordinated Cellular Neighborhoods Orchestrate Antitumoral Immunity
at the Colorectal Cancer Invasive Front. Cell. 2020
Sep;182(5):1341-1359.e19.</p>
<p>118. Haviv D, Remšík J, Gatie M, Snopkowski C, Takizawa M, Pereira N,
et al. The covariance environment defines cellular niches for spatial
inference. Nat Biotechnol. 2025 Feb;43(2):269--80.</p>
<p>119. Birk S, Bonafonte-Pardàs I, Feriz AM, Boxall A, Agirre E, Memi F,
et al. Quantitative characterization of cell niches in spatially
resolved omics data. Nat Genet. 2025 Apr;57(4):897--909.</p>
<p>120. Zhao E, Stone MR, Ren X, Guenthoer J, Smythe KS, Pulliam T, et al.
Spatial transcriptomics at subspot resolution with BayesSpace. Nat
Biotechnol. 2021 Nov;39(11):1375--84.</p>
<p>121. Zhu Q, Shah S, Dries R, Cai L, Yuan GC. Identification of
spatially associated subpopulations by combining scRNAseq and sequential
fluorescence in situ hybridization data. Nat Biotechnol. 2018
Dec;36(12):1183--90.</p>
<p>122. Pham D, Tan X, Balderson B, Xu J, Grice LF, Yoon S, et al. Robust
mapping of spatiotemporal trajectories and cell--cell interactions in
healthy and diseased tissues. Nat Commun. 2023 Nov 25;14(1):7739.</p>
<p>123. Hu J, Li X, Coleman K, Schroeder A, Ma N, Irwin DJ, et al. SpaGCN:
Integrating gene expression, spatial location and histology to identify
spatial domains and spatially variable genes by graph convolutional
network. Nat Methods. 2021 Nov;18(11):1342--51.</p>
<p>124. Dong K, Zhang S. Deciphering spatial domains from spatially
resolved transcriptomics with an adaptive graph attention auto-encoder.
Nat Commun. 2022 Apr 1;13(1):1739.</p>
<p>125. Long Y, Ang KS, Li M, Chong KLK, Sethi R, Zhong C, et al.
Spatially informed clustering, integration, and deconvolution of spatial
transcriptomics with GraphST. Nat Commun. 2023 Mar 1;14(1):1155.</p>
<p>126. Andersson A, Behanova A, Avenel C, Windhager J, Malmberg F, Wählby
C. [Points2Regions]{.smallcaps} : Fast, interactive clustering of
imaging‐based spatial transcriptomics data. Cytometry A. 2024
Sep;105(9):677--87.</p>
<p>127. Walker BL, Nie Q. NeST: nested hierarchical structure
identification in spatial transcriptomic data. Nat Commun. 2023 Oct
17;14(1):6554.</p>
<p>128. Bhate SS, Barlow GL, Schürch CM, Nolan GP. Tissue schematics map
the specialization of immune tissue motifs and their appropriation by
tumors. Cell Syst. 2022 Feb;13(2):109-130.e6.</p>
<p>129. Getis A. Spatial Autocorrelation. In: Fischer MM, Getis A,
editors. Handbook of Applied Spatial Analysis [Internet]. Berlin,
Heidelberg: Springer Berlin Heidelberg; 2010 [cited 2025 Apr 20]. p.
255--78. Available from:
https://link.springer.com/10.1007/978-3-642-03647-7_14</p>
<p>130. Svensson V, Teichmann SA, Stegle O. SpatialDE: identification of
spatially variable genes. Nat Methods. 2018 May;15(5):343--6.</p>
<p>131. Edsgärd D, Johnsson P, Sandberg R. Identification of spatial
expression trends in single-cell gene expression data. Nat Methods. 2018
May;15(5):339--42.</p>
<p>132. Sun S, Zhu J, Zhou X. Statistical analysis of spatial expression
patterns for spatially resolved transcriptomic studies. Nat Methods.
2020 Feb;17(2):193--200.</p>
<p>133. Zhu J, Sun S, Zhou X. SPARK-X: non-parametric modeling enables
scalable and robust detection of spatial expression patterns for large
spatial transcriptomic studies. Genome Biol. 2021 Dec;22(1):184.</p>
<p>134. Andersson A, Lundeberg J. <em>sepal</em> : identifying transcript
profiles with spatial patterns by diffusion-based modeling. Mathelier A,
editor. Bioinformatics. 2021 Sep 9;37(17):2644--50.</p>
<p>135. Zhang K, Feng W, Wang P. Identification of spatially variable
genes with graph cuts. Nat Commun. 2022 Sep 19;13(1):5488.</p>
<p>136. Armingol E, Baghdassarian HM, Lewis NE. The diversification of
methods for studying cell--cell interactions and communication. Nat Rev
Genet. 2024 Jun;25(6):381--400.</p>
<p>137. Dimitrov D, Schäfer PSL, Farr E, Rodriguez-Mier P, Lobentanzer S,
Badia-i-Mompel P, et al. LIANA+ provides an all-in-one framework for
cell--cell communication inference. Nat Cell Biol. 2024
Sep;26(9):1613--22.</p>
<p>138. Fischer DS, Schaar AC, Theis FJ. Modeling intercellular
communication in tissues using spatial graphs of cells. Nat Biotechnol.
2023 Mar;41(3):332--6.</p>
<p>139. Troulé K, Petryszak R, Cakir B, Cranley J, Harasty A, Prete M, et
al. CellPhoneDB v5: inferring cell--cell communication from single-cell
multiomics data. Nat Protoc [Internet]. 2025 Mar 25 [cited 2025 Apr
20]; Available from: https://www.nature.com/articles/s41596-024-01137-1</p>
<p>140. Jin S, Plikus MV, Nie Q. CellChat for systematic analysis of
cell--cell communication from single-cell transcriptomics. Nat Protoc.
2025 Jan;20(1):180--219.</p>
<p>141. Shao X, Li C, Yang H, Lu X, Liao J, Qian J, et al.
Knowledge-graph-based cell-cell communication inference for spatially
resolved transcriptomic data with SpaTalk. Nat Commun. 2022 Jul
30;13(1):4429.</p>
<p>142. Cang Z, Zhao Y, Almet AA, Stabell A, Ramos R, Plikus MV, et al.
Screening cell--cell communication in spatial transcriptomics via
collective optimal transport. Nat Methods. 2023 Feb;20(2):218--28.</p>
<p>143. Li Z, Wang T, Liu P, Huang Y. SpatialDM for rapid identification
of spatially co-expressed ligand--receptor and revealing cell--cell
communication patterns. Nat Commun. 2023 Jul 6;14(1):3995.</p>
<p>144. Browaeys R, Saelens W, Saeys Y. NicheNet: modeling intercellular
communication by linking ligands to target genes. Nat Methods. 2020 Feb
2;17(2):159--62.</p>
<p>145. Yuan Y, Bar-Joseph Z. GCNG: graph convolutional networks for
inferring gene interaction from spatial transcriptomics data. Genome
Biol. 2020 Dec;21(1):300.</p>
<p>146. Zhu J, Wang Y, Chang WY, Malewska A, Napolitano F, Gahan JC, et
al. Mapping cellular interactions from spatially resolved
transcriptomics data. Nat Methods. 2024 Oct;21(10):1830--42.</p>
<p>147. Walker BL, Cang Z, Ren H, Bourgain-Chang E, Nie Q. Deciphering
tissue structure and function using spatial transcriptomics. Commun
Biol. 2022 Mar 10;5(1):220.</p>
<p>148. Türei D, Korcsmáros T, Saez-Rodriguez J. OmniPath: guidelines and
gateway for literature-curated signaling pathway resources. Nat Methods.
2016 Dec;13(12):966--7.</p>
<p>149. Arnol D, Schapiro D, Bodenmiller B, Saez-Rodriguez J, Stegle O.
Modeling Cell-Cell Interactions from Spatial Molecular Data with Spatial
Variance Component Analysis. Cell Rep. 2019 Oct;29(1):202-211.e6.</p>
<p>150. Tanevski J, Flores ROR, Gabor A, Schapiro D, Saez-Rodriguez J.
Explainable multiview framework for dissecting spatial relationships
from highly multiplexed data. Genome Biol. 2022 Dec;23(1):97.</p>
<p>151. Badia-i-Mompel P, Vélez Santiago J, Braunger J, Geiss C, Dimitrov
D, Müller-Dott S, et al. decoupleR: ensemble of computational methods to
infer biological activities from omics data. Kuijjer ML, editor.
Bioinforma Adv. 2022 Jan 10;2(1):vbac016.</p>
<p>152. Schubert M, Klinger B, Klünemann M, Sieber A, Uhlitz F, Sauer S,
et al. Perturbation-response genes reveal signaling footprints in cancer
gene expression. Nat Commun. 2018 Jan 2;9(1):20.</p>
<p>153. Müller-Dott S, Tsirvouli E, Vazquez M, Ramirez Flores RO,
Badia-i-Mompel P, Fallegger R, et al. Expanding the coverage of regulons
from high-confidence prior knowledge for accurate estimation of
transcription factor activities. Nucleic Acids Res. 2023 Nov
10;51(20):10934--49.</p>
<p>154. Palla G, Spitzer H, Klein M, Fischer D, Schaar AC, Kuemmerle LB,
et al. Squidpy: a scalable framework for spatial omics analysis. Nat
Methods. 2022 Feb;19(2):171--8.</p>
<p>155. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC,
et al. Multi‐Omics Factor Analysis---a framework for unsupervised
integration of multi‐omics data sets. Mol Syst Biol. 2018
Jun;14(6):e8124.</p>
<p>156. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni
JC, et al. MOFA+: a statistical framework for comprehensive integration
of multi-modal single-cell data. Genome Biol. 2020 Dec;21(1):111.</p>
<p>157. Ramirez Flores RO, Lanzer JD, Dimitrov D, Velten B, Saez-Rodriguez
J. Multicellular factor analysis of single-cell data for a
tissue-centric understanding of disease. eLife. 2023 Nov 22;12:e93161.</p>
<p>158. Velten B, Braunger JM, Argelaguet R, Arnol D, Wirbel J, Bredikhin
D, et al. Identifying temporal and spatial patterns of variation from
multimodal data using MEFISTO. Nat Methods. 2022 Feb;19(2):179--86.</p>
<p>159. Jerby-Arnon L, Regev A. DIALOGUE maps multicellular programs in
tissue from single-cell or spatial transcriptomics data. Nat Biotechnol.
2022 Oct;40(10):1467--77.</p>
<p>160. Mitchel J, Gordon MG, Perez RK, Biederstedt E, Bueno R, Ye CJ, et
al. Coordinated, multicellular patterns of transcriptional variation
that stratify patient cohorts are revealed by tensor decomposition. Nat
Biotechnol [Internet]. 2024 Sep 23 [cited 2025 Apr 20]; Available
from: https://www.nature.com/articles/s41587-024-02411-z</p>
<p>161. Townes FW, Engelhardt BE. Nonnegative spatial factorization
applied to spatial genomics. Nat Methods. 2023 Feb;20(2):229--38.</p>
<p>162. Jiménez-Sánchez D, Ariz M, Chang H, Matias-Guiu X, De Andrea CE,
Ortiz-de-Solórzano C. NaroNet: Discovery of tumor microenvironment
elements from highly multiplexed images. Med Image Anal. 2022
May;78:102384.</p>
<p>163. Fu X, Patrick E, Yang JYH, Feng DD, Kim J. Deep multimodal
graph-based network for survival prediction from highly multiplexed
images and patient variables. Comput Biol Med. 2023 Mar;154:106576.</p>
<p>164. Risom T, Glass DR, Averbukh I, Liu CC, Baranski A, Kagel A, et al.
Transition to invasive breast cancer is associated with progressive
changes in the structure and composition of tumor stroma. Cell. 2022
Jan;185(2):299-310.e18.</p>
<p>165. Babaei S, Christ J, Sehra V, Makky A, Zidane M, Wistuba-Hamprecht
K, et al. S3-CIMA: Supervised spatial single-cell image analysis for
identifying disease-associated cell-type compositions in tissue.
Patterns. 2023 Sep;4(9):100829.</p>
<p>166. Wang Y, Wang YG, Hu C, Li M, Fan Y, Otter N, et al. Cell graph
neural networks enable the precise prediction of patient survival in
gastric cancer. Npj Precis Oncol. 2022 Jun 23;6(1):45.</p>
<p>167. Perez-Lopez R, Ghaffari Laleh N, Mahmood F, Kather JN. A guide to
artificial intelligence for cancer researchers. Nat Rev Cancer. 2024
Jun;24(6):427--41.</p>
<p>168. Unger M, Kather JN. A systematic analysis of deep learning in
genomics and histopathology for precision oncology. BMC Med Genomics.
2024 Feb 5;17(1):48.</p>
<p>169. Hao Y, Hao S, Andersen-Nissen E, Mauck WM, Zheng S, Butler A, et
al. Integrated analysis of multimodal single-cell data. Cell. 2021
Jun;184(13):3573-3587.e29.</p>
<p>170. Wolf FA, Angerer P, Theis FJ. SCANPY: large-scale single-cell gene
expression data analysis. Genome Biol. 2018 Dec;19(1):15.</p>
<p>171. Virshup I, Rybakov S, Theis FJ, Angerer P, Wolf FA. anndata:
Access and store annotated datamatrices. J Open Source Softw. 2024 Sep
16;9(101):4371.</p>
<p>172. Dries R, Zhu Q, Dong R, Eng CHL, Li H, Liu K, et al. Giotto: a
toolbox for integrative analysis and visualization of spatial expression
data. Genome Biol. 2021 Dec;22(1):78.</p>
<p>173. Marconato L, Palla G, Yamauchi KA, Virshup I, Heidari E, Treis T,
et al. SpatialData: an open and universal data framework for spatial
omics. Nat Methods. 2025 Jan;22(1):58--62.</p>
<p>174. Sofroniew N, Lambert T, Bokota G, Nunez-Iglesias J, Sobolewski P,
Sweet A, et al. napari: a multi-dimensional image viewer for Python
[Internet]. Zenodo; 2025 [cited 2025 Apr 14]. Available from:
https://zenodo.org/doi/10.5281/zenodo.3555620</p>
<p>175. Roesch E, Greener JG, MacLean AL, Nassar H, Rackauckas C, Holy TE,
et al. Julia for biologists. Nat Methods. 2023 May;20(5):655--64.</p>
<p>176. Virshup I, Bredikhin D, Heumos L, Palla G, Sturm G, Gayoso A, et
al. The scverse project provides a computational ecosystem for
single-cell omics data analysis. Nat Biotechnol. 2023 May;41(5):604--6.</p>
<p>177. Liu CC, Greenwald NF, Kong A, McCaffrey EF, Leow KX, Mrdjen D, et
al. Robust phenotyping of highly multiplexed tissue imaging data using
pixel-level clustering. Nat Commun. 2023 Aug 1;14(1):4618.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.tabs", "navigation.sections", "toc.integrate"], "search": "../../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.13a4f30d.min.js"></script>
      
    
  </body>
</html>